irods_testing_environment/test_runner.py

```diff
@@ -130,12 +130,12 @@ def run(self, fail_fast=True, *args):
 
             if ec is 0:
                 self.passed_tests()[t] = duration
-                logging.info('[{}]: cmd succeeded [{}]'.format(self.name(), cmd))
+                logging.error('[{}]: cmd succeeded [{}]'.format(self.name(), cmd))
```

Comment:
yeah, weird this is error.   but i understand the reasoning.

Reply:
Ja. See #41

Reply:
oh, that's a smart conversation.    so smart.